src/main/java/com/o19s/es/ltr/action/CachesStatsAction.java

```diff
@@ -39,7 +39,7 @@
 
 public class CachesStatsAction extends Action<CachesStatsAction.CachesStatsNodesRequest,
         CachesStatsAction.CachesStatsNodesResponse, CachesStatsAction.CacheStatsRequestBuilder> {
-    public static final String NAME = "ltr:caches/stats";
+    public static final String NAME = "indices:data/ltr/caches/stats";
```

Comment:
Cache stats are global, and we don't report them per-index. Maybe this should be a cluster admin action? not sure.